pages/lessons/[name].js

```diff
@@ -0,0 +1,98 @@
+import styled from "styled-components";
+import { getLessonByName, getAllLessons } from "../../lib/db";
+import Image from "next/image";
+
+export async function getStaticPaths() {
+  const lessons = await getAllLessons();
```

Comment:
Then you also don't need the await here :)

Reply:
Hi Cathrin, thanks for the feedback! I already had the DB mirgration in mind the moment I wrote the code. Therefore I just wrote the reason in my commit message :)
[change static and db function to async in preparation of db migration] [9700a55]

Reply:
Ah ok, I see. Then everything looks good to me :) Nice code!